code/game/objects/items/weapons/nt_melee.dm

```diff
@@ -96,19 +136,114 @@
 		icon_state = initial(icon_state)
 	..()
 
+/obj/item/weapon/tool/sword/nt/scourge/apply_hit_effect(mob/living/carbon/human/target, mob/living/user, hit_zone)
+	..()
+	if(ishuman(target))
+		var/mob/living/carbon/human/O = target
+		target.stun_effect_act(stun, agony, hit_zone, src)
+		O.say(pick("OH", "LORD", "MERCY", "SPARE", "ME", "HAVE", "PLEASE"))
+
+/obj/item/weapon/tool/sword/nt/spear
+	name = "NT Pilum"
+	desc = "A saint looking short spear, designed for use with a shield or as a throwing weapon. The spear-tip usually breaks after being thrown at a target, but it can be welded into shape again."
+	icon_state = "nt_spear"
+	item_state = "nt_spear"
+	wielded_icon = "nt_spear_wielded"
+	force = 24
+	force_wielded_multiplier = 1.08
+	var/tipbroken = FALSE
+	w_class = ITEM_SIZE_HUGE
+	slot_flags = SLOT_BACK | SLOT_BELT 
+	throwforce = 75
+	armor_penetration = ARMOR_PEN_HALF
+	throw_speed = 3
+	price_tag = 150
+	allow_spin = FALSE
+	matter = list(MATERIAL_BIOMATTER = 10, MATERIAL_STEEL = 5) // easy to mass-produce and arm the faithful
+
+/obj/item/weapon/tool/sword/nt/spear/equipped(mob/living/W)
+	..()
+	if(is_held() && is_neotheology_disciple(W))
+		embed_mult = 0.1
+	else
+		embed_mult = initial(embed_mult)
+
+/obj/item/weapon/tool/sword/nt/spear/dropped(mob/living/W)
+	embed_mult = 300
+	..()
+
+/obj/item/weapon/tool/sword/nt/spear/on_embed(mob/user)
+	..()
+	tipbroken = TRUE
```

Comment:
```suggestion
	if(ismob(hit_atom)
		tipbroken = TRUE
```
to prevent it from breaking outside of combat, all you need to do is add this, yes?

Reply:
This wouldn't change anything since on_embed is only called upon embedding anyways.